getting_started_GPIOs/GPIO_01_Shell/README.md

```diff
@@ -0,0 +1,73 @@
+# General Purpose Input/Output
```

Comment:
Please split the home surveillance project updates into separate commit

Reply:
Hi Mani,

I understand that we have guides section in documentation repo.
The purpose of having this getting started with GPIO's within the projects folder is to help with hackathon events. We have noticed that participants are on a time crunch during these events and in most cases they don't do enough research and end up with wrong resources. 
We will be participating in Major League Hacks (MLH) this year, so there will be about 200 DragonBoards shipped to various universities in US that will be used for Hacks. As part of the process, we are having the MLH staff re-flash the DragonBoards and git clone the projects repository on them as a reference.
This will help us leverage the awesome work that you guys have put in to the projects repo. Adding the getting started with GPIO's will keep everything consolidated for the Hackathon participants.

Reply:
Sounds good!

Thanks Rajan.